drivers/serial/ns16550a/config.json

```diff
@@ -1,6 +1,7 @@
 {
     "compatible": [
-        "starfive,jh7110-uart"
+        "starfive,jh7110-uart",
```

Comment:
Should we do `snps,dw-apb-uart` now? The Star64 seems to have `compatible = "starfive,jh7110-uart", "snps,dw-apb-uart";`.

Reply:
Will keep it as it is now, it's very easy to change in the future if we have another board that needs `"snps,dw-apb-uart"`